### PR TITLE
Add section about configuration files

### PR DESCRIPTION
**Pull request by dabcoder**
### What does this PR do?
Add a section about mounting configuration files. This is also described here: https://github.com/DataDog/docker-dd-agent#configuration-files

### Motivation
Add instructions in the documentation for better clarity.

### Preview link
https://github.com/DataDog/documentation/blob/davidb/docker-mount-conf/content/agent/basic_agent_usage/docker.md#configuration-files

### CODE REVIEW

File: content/agent/basic_agent_usage/docker.md [hunk 1]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
 
 **Note**: The `docker.containers.running`, `.stopped`, `.running.total` and `.stopped.total` metrics are not affected by these settings and always count all containers. This does not affect your per-container billing.
 
+### Configuration files
+
+You can also mount YAML configuration files in the `/conf.d` folder, they will automatically be copied to `/etc/dd-agent/conf.d/` when the container starts.  The same can be done for the `/checks.d` folder. Any Python files in the `/checks.d` folder will automatically be copied to `/etc/dd-agent/checks.d/` when the container starts.
```
Comment (antoinepouille) [inline content/agent/basic_agent_usage/docker.md]: dd-agent -> datadog-agent for agent 6.

File: content/agent/basic_agent_usage/docker.md [hunk 2]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
...
+
+1. Create a configuration folder on the host and write your YAML files in it.  The examples below can be used for the `/checks.d` folder as well.
+
+    ```
+    mkdir /opt/dd-agent-conf.d
+    touch /opt/dd-agent-conf.d/nginx.yaml
```
Comment (antoinepouille) [inline content/agent/basic_agent_usage/docker.md]: Same here

File: content/agent/basic_agent_usage/docker.md [hunk 3]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
...
+
+2. When creating the container, mount this new folder to `/conf.d`.
+    ```
+    docker run -d --name dd-agent \
+      -v /var/run/docker.sock:/var/run/docker.sock:ro \
+      -v /proc/:/host/proc/:ro \
+      -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+      -v /opt/dd-agent-conf.d:/conf.d:ro \
+      -e API_KEY={your_api_key_here} \
+      datadog/docker-dd-agent
```
Comment (antoinepouille) [inline content/agent/basic_agent_usage/docker.md]: Same here, you can take the example for the top of the file, which is for agent 6 :-)

File: content/agent/basic_agent_usage/docker.md [hunk 4]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
...
+    ```
+
+    _The important part here is `-v /opt/dd-agent-conf.d:/conf.d:ro`_
```
Comment (antoinepouille) [inline content/agent/basic_agent_usage/docker.md]: ditto

File: content/agent/basic_agent_usage/docker.md [hunk 5]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
...
+
+Now when the container starts, all files in `/opt/dd-agent-conf.d` with a `.yaml` extension will be copied to `/etc/dd-agent/conf.d/`. Please note that to add new files you will need to restart the container.
```
Comment (antoinepouille) [inline content/agent/basic_agent_usage/docker.md]: ditto

**Comment by dabcoder** [comment]
@antoinepouille updated.

File: content/agent/basic_agent_usage/docker.md [hunk 6]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
 
 **Note**: The `docker.containers.running`, `.stopped`, `.running.total` and `.stopped.total` metrics are not affected by these settings and always count all containers. This does not affect your per-container billing.
 
+### Configuration files
+
+You can also mount YAML configuration files in the `/conf.d` folder, they will automatically be copied to `/etc/datadog-agent/conf.d/` when the container starts.  The same can be done for the `/checks.d` folder. Any Python files in the `/checks.d` folder will automatically be copied to `/etc/datadog-agent/checks.d/` when the container starts.
```
Comment (cswatt) [inline content/agent/basic_agent_usage/docker.md]: Some spacing and punctuation.

You can also mount YAML configuration files in the `/conf.d` folder. They will automatically be copied to `/etc/datadog-agent/conf.d/` when the container starts. The same can be done for the `/checks.d` folder: any Python files in the `/checks.d` folder will automatically be copied to `/etc/datadog-agent/checks.d/` when the container starts.

File: content/agent/basic_agent_usage/docker.md [hunk 7]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
...
+
+1. Create a configuration folder on the host and write your YAML files in it.  The examples below can be used for the `/checks.d` folder as well.
```
Comment (cswatt) [inline content/agent/basic_agent_usage/docker.md]: One space between sentences.

File: content/agent/basic_agent_usage/docker.md [hunk 8]
```diff
@@ -92,6 +92,32 @@ Exclude containers from the metrics collection and Autodiscovery, if these are n
...
+      -v /proc/:/host/proc/:ro \
+      -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+      -v /opt/datadog-agent-conf.d:/conf.d:ro \
+      -e API_KEY={your_api_key_here} \
+       datadog/agent:latest
+    ```
+
+    _The important part here is `-v /opt/datadog-agent-conf.d:/conf.d:ro`_
+
+Now when the container starts, all files in `/opt/datadog-agent-conf.d` with a `.yaml` extension will be copied to `/etc/datadog-agent/conf.d/`. Please note that to add new files you will need to restart the container.
```
Comment (cswatt) [inline content/agent/basic_agent_usage/docker.md]: Doc style guide recommends against usage of the word "please."

**Comment by antoinepouille** [comment]
LGTM content-wise

**Pull request merged**